### PR TITLE
fixes tutorial issue 150

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes https://github.com/Project-MONAI/tutorials/issues/150

### Description
added some 2D testing cases

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] New tests added to cover the changes.

### CODE REVIEW

File: monai/transforms/spatial/dictionary.py [hunk 1]
```diff
@@ -1285,7 +1285,7 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
         d = dict(data)
         if not self._do_transform:
             for key in self.keys:
-                self.push_transform(d, key, extra_info={"rot_mat": np.eye(4)})
+                self.push_transform(d, key, extra_info={"rot_mat": np.eye(d[key].ndim)})
```
Comment (rijobro) [inline monai/transforms/spatial/dictionary.py:1288]: Should this be ndim+1? Not sure if it's purely rotation matrix or also translation component.

Comment (rijobro) [reply monai/transforms/spatial/dictionary.py:1288]: Sorry, thought about this a bit more, and it makes sense.

File: tests/test_inverse_collation.py [hunk 2]
```diff
@@ -53,6 +53,19 @@
...
 
+TESTS_2D = [
+    (t.__class__.__name__ + (" pad_list_data_collate" if collate_fn else " default_collate"), t, collate_fn, 2)
+    for collate_fn in [None, pad_list_data_collate]
+    for t in [
+        RandFlipd(keys=KEYS, spatial_axis=[1]),
+        RandAxisFlipd(keys=KEYS),
+        RandRotate90d(keys=KEYS, spatial_axes=(0, 1)),
+        RandZoomd(keys=KEYS, prob=0.5, min_zoom=0.5, max_zoom=1.1, keep_size=True),
+        RandRotated(keys=KEYS, range_x=np.pi),
```
Comment (rijobro) [inline tests/test_inverse_collation.py:64]: By default probability is 0.1, so for batch size of 7, there's no guarantee of a mix of `True` and `False` for `_do_transform`. Should probably just set `prob=0.5` for all transforms.

**Review by rijobro — APPROVED** [review]

File: monai/transforms/spatial/dictionary.py (hunk 1 above)
Comment (wyli) [reply monai/transforms/spatial/dictionary.py:1288]: yes it's with the channel dimension...

File: tests/test_inverse_collation.py (hunk 2 above)
Comment (wyli) [reply tests/test_inverse_collation.py:64]: good point, I'll revise...

**Pull request merged**